doc/unpackGRAW.rst

```diff
@@ -0,0 +1,59 @@
+1) Merge and Unpack GRAW Files into HDF5 Files
+==============================================
+
+Installing and Compiling
+------------------------
+
+**This step in the analysis should be performed locally.** Install the graw-merger tool and it's dependencies. The graw-merger repository contains the source code for the `graw2hdf` tool, which can be used to merge the GRAW files produced by the GET electronics into an `HDF5 <https://www.hdfgroup.org/HDF5/>`__ file. Armadillo should already be installed, but some `Boost C++ libraries <http://www.boost.org/>`__ are required for this step. Use wget to download to download the .tar archive and extract it (find the correct download `here <http://www.boost.org/users/download/>`__):
+
+.. note::
+   
+   You may wish to use Homebrew to install the `boost-compute` package and skips these steps if running macOS.
```

Comment:
Perhaps also suggest using the package manager on Linux. Building Boost is a pain.